src/makeRobot.js

```diff
@@ -38,7 +38,58 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    get info() {
```

Comment:
would be better to add new line between function blocks
```suggestion
    version,

    get info() {
```